classes/container.php

```diff
@@ -28,7 +28,12 @@ public function build(atoum\scripts\runner $script)
 
         $loader = new YamlFileLoader($this->container, new FileLocator(array(__DIR__ . '/../resources', getcwd())));
         $loader->load('services.yml');
-        $loader->load('.atoum.yml');
+
+        try {
+            $loader->load('.atoum.yml');
```

Comment:
ok. I will ll change that. 
but I will also try to remove the filelocator for the .atoum.yml, just to be certain that the check and the loaded file are on the same path (for now the .atoum.yml could also be in the resources folder).